src/main/java/com/example/example_mod/ExampleMod.java

```diff
@@ -1,15 +1,18 @@
-package com.examplemod;
+package com.example.example_mod;
 
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.quiltmc.loader.api.ModContainer;
 import org.quiltmc.qsl.base.api.entrypoint.ModInitializer;
 
 public class ExampleMod implements ModInitializer {
-	public static final Logger LOGGER = LogManager.getLogger("ExampleMod");
+	// This logger is used to write text to the console and the log file.
+	// It is considered best practice to use your mod name as the logger's name.
+	// That way, it's clear which mod wrote info, warnings, and errors.
+	public static final Logger LOGGER = LogManager.getLogger("Example Mod");
 
 	@Override
 	public void onInitialize(ModContainer mod) {
-		LOGGER.info( "Hello Quilt world from ExampleMod v{}!", mod.metadata().version().raw() );
+		LOGGER.info("Hello Quilt world from Example Mod v{}!", mod.metadata().version().raw());
```

Comment:
Actually, could we maybe make this use `mod.metadata().name().raw()`? :lil_pineapple:
```suggestion
		LOGGER.info("Hello Quilt world from {} v{}!", mod.metadata().name().raw(), mod.metadata().version().raw());
```